control/tests/timeresp_test.py

```diff
@@ -270,6 +270,17 @@ def test_forced_response(self):
         np.testing.assert_array_equal(youtc.shape, youtd.shape)
         np.testing.assert_array_almost_equal(youtc, youtd, decimal=4)
 
+        # Test discrete MIMO system without default T argument
+        u = np.array([[0., 0, 0, 0, 0, 0, 0, 0, 0, 0],
+                      [1., 1, 1, 1, 1, 1, 1, 1, 1, 1]])
+        x0 = np.matrix(".5; 1; 0; 0")
```

Comment:
`np.matrix` is being deprecated.  Use `np.array` instead to avoid creating warning messages.

Reply:
Switched to np.matrix in updated pull request